docs/installation/installation.rst

```diff
@@ -297,6 +330,10 @@ either of the following methods:
 
    .. tab:: Windows (WSL2)
 
+      .. warning::
+         Support for installing :program:`MIRACL` locally in the :program:`WSL`
+         has been deprecated in version ``2.2.6`` of :program:`MIRACL`.
+
```

Comment:
What was the reason we stopped supporting WSL installation? Isn't that the only way to install it on windows? 
Or they would use apptainer instead?

Reply:
We still support installing MIRACL in the WSL but not locally anymore, as in installing MIRACL step by step in the WSL directly instead of using Docker in the WSL. Windows users should use the WSL but will still need Docker to install MIRACL in it. Do you think this part of the docs could be confusing to ppl?

Reply:
Yep that is right. It may be confusing because it confused me for a second :laughing: . Maybe right after we say local is depreciated we can say something like "MIRACL can only be installed in WSL using Docker". What do you think?

Reply:
Done - sorry for the delay.